examples/redis_key_source.rs

```diff
@@ -0,0 +1,53 @@
+use config::builder::AsyncState;
+use redis::AsyncCommands;
+use redis_config::RedisSource;
+
+const REDIS_URL: &str = "redis://127.0.0.1:6379";
+const SOURCE_KEY: &str = "application-settings";
```

Comment:
Seems it's better to use a flexible configuration file with the ability to use different APP_ENV configs and env variables.

Reply:
Added an Env source in the examples/ directory

Reply:
Awesome!!!